src/evict/evict_file.c

```diff
@@ -6,7 +6,8 @@
  * See the file LICENSE for redistribution information.
  */
 
-#include "wt_internal.h"
+#include "evict.h"
```

Comment:
eviction's files no longer need wt_internal, just their public and private interfaces.